module/data/activity/base-activity.mjs

```diff
@@ -162,7 +165,10 @@ export default class BaseActivityData extends foundry.abstract.DataModel {
    * @type {ActiveEffect5e[]|null}
    */
   get applicableEffects() {
```

Comment:
Are we already filtering out rider effects somewhere?

Reply:
This is only looking at the effects defined in the activity, so we do that filter in the application when we provide the list of effects to choose from.